upi/openstack/security-groups.yaml

```diff
@@ -344,6 +344,42 @@
         port_range_min: 443
         port_range_max: 443
 
+    - name: 'Create master-sg IPv6 rule "master ingress services (TCP)"'
+      openstack.cloud.security_group_rule:
+        security_group: "{{ os_sg_master }}"
+        ethertype: IPv6
+        protocol: tcp
+        remote_ip_prefix: "{{ os_subnet6_range }}"
+        port_range_min: 30000
+        port_range_max: 32767
+
+    - name: 'Create master-sg IPv6 rule "master ingress services (UDP)"'
+      openstack.cloud.security_group_rule:
+        security_group: "{{ os_sg_master }}"
+        ethertype: IPv6
+        protocol: udp
+        remote_ip_prefix: "{{ os_subnet6_range }}"
+        port_range_min: 30000
+        port_range_max: 32767
+
+    - name: 'Create worker-sg IPv6 rule "worker ingress services (TCP)"'
+      openstack.cloud.security_group_rule:
+        security_group: "{{ os_sg_worker }}"
+        ethertype: IPv6
+        protocol: tcp
+        remote_ip_prefix: "{{ os_subnet6_range }}"
+        port_range_min: 30000
+        port_range_max: 32767
+
+    - name: 'Create worker-sg rule IPv6 "worker ingress services (UDP)"'
+      openstack.cloud.security_group_rule:
+        security_group: "{{ os_sg_worker }}"
+        ethertype: IPv6
+        protocol: udp
+        remote_ip_prefix: "{{ os_subnet6_range }}"
+        port_range_min: 30000
+        port_range_max: 32767
+
```

Comment:
Could we not split the 80&443 rules of masters and workers and place this below them? Then it's `/lgtm`

Reply:
done